sos/report/plugins/openstack_edpm.py

```diff
@@ -24,7 +24,10 @@ def setup(self):
         # Notes: recursion is max 2 for edpm-config
         # Those directories are present on all OpenStack nodes
         self.edpm_log_paths = [
-            '/var/lib/edpm-config/'
+            '/etc/os-net-config/',
+            '/var/lib/config-data/',
+            '/var/lib/edpm-config/',
+            '/var/log/containers/',
```

Comment:
Nack to `/var/log/containers` as a whole. There are _tons_ of components that drop logs there these days.

The `container_log` plugin capture this as needed, but if you need specific containers call them out explicitly in these paths, e.g. `/var/log/containers/$container/$log` which is what many of the other openstack plugins do.

Reply:
dropped /var/log/containers